db/seeds.rb

```diff
@@ -1,7 +1,21 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+if Rails.env.development?
+
+  2.times do |i|
+    # Users
+    u = User.create!(
+                      username: Faker::Internet.user_name, 
+                      email: Faker::Internet.safe_email, 
+                      token: Faker::Internet.password(20)
+                    )
+    rand(10..20).times do |j|
+      # Gems
+      ruby_gem = RubyGem.create!(
+                                  name: "#{Faker::App.name}_#{i}_#{j}", 
+                                  info: Faker::Lorem.paragraph(2), 
+                                  project_uri: Faker::Internet.url('example.com')
+                                )
+      u.ruby_gems << ruby_gem
+      u.gems_adoptions.create!(ruby_gem_id: ruby_gem.id, description: Faker::Lorem.paragraph)
+    end
+  end
```

Comment:
I think this can be useful, but I'm not sure having fake gems would be all that useful. Maybe we could ask the RubyGems.org team how they handle populating their local database for development.

I'd rather not introduce a dependency to the project to create seeds, I think you can achieve all of this, by adding static values where possible and using the counter where values need to be unique.

I think this is not really related to adding RubyGems.org look and feel and should be added in another pull request where we can better discuss how to approach seeds.